src/redisearch_rs/c_entrypoint/inverted_index_ffi/build.rs

```diff
@@ -17,6 +17,7 @@ fn main() {
 
     let config = Config::from_file("cbindgen.toml").expect("Failed to find cbindgen config");
     let _ = rerun_cbinden(&config);
+    println!("cargo::rerun-if-changed=cbindgen.toml");
```

Comment:
Would it make sense to add this to `rerun_cbinden()` ?

Reply:
Hmmm, at first I thought no because you can't access the config file path from the `&config` param, but I guess there is no harm in passing it as a string 🤔